common/history/countries/den - denmark.txt

```diff
@@ -8,7 +8,7 @@
 		# Laws
 		effect_create_law_package = {
 			headOfGovernment = hereditary
-			distributionOfPower = representative_democracy
+			distributionOfPower = consensus_democracy
```

Comment:
Most nations don't start with the tech for consensus democracy u less this requirement was changed